core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -164,4 +164,11 @@ private TableProperties() {
 
   public static final String MERGE_MODE = "write.merge.mode";
   public static final String MERGE_MODE_DEFAULT = "copy-on-write";
+
+  public static final String MERGE_WRITE_CARDINALITY_CHECK = "write.merge.cardinality-check.enabled";
+  public static final boolean MERGE_WRITE_CARDINALITY_CHECK_DEFAULT = true;
+
+  public static final String MERGE_WRITE_SORT_MODE = "write.merge.sort.mode";
```

Comment:
We should consider this with the [context from the thread about Flink write distribution](https://github.com/apache/iceberg/pull/2064#discussion_r559018257).